### PR TITLE
Ar 864 new player - merge master

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -22,6 +22,7 @@
     f.parentNode.insertBefore(j, f);
   })(window, document, 'script', 'dataLayer', 'GTM-5WM8NVH');</script>
   <!-- End Google Tag Manager -->
+  <script async src="https://cdn.jwplayer.com/libraries/mxNkRalL.js"></script>
```
Comment (bbfsdev) [inline public/index.html:25]: Do we want to be dependent on external resource?

File: public/locales/en/common.json [hunk 2]
```diff
@@ -176,24 +176,49 @@
...
       "series-by-topic": "Series of lessons on the topic:"
     },
     "download": {
-      "iconHoverText": "Download the excerpt you marked",
-      "downloadButton": "Download",
+      "icon-hover-text": "Download the excerpt you marked",
+      "download-button": "Download",
       "copyButton": "Copy link"
+    },
+    "settings": {
```
Comment (bbfsdev) [inline public/locales/en/common.json:192]: Are all those properly updated in all languages?

File: src/components/Layout/Layout.js [hunk 3]
```diff
@@ -225,10 +225,14 @@ class Layout extends Component {
                       </Menu.Item>
                     </Ref>
                   }
-                  <Menu.Item position="right" className="mobile-hidden">
-                    <DonateNow language={language} />
-                    <VirtualHomeButton language={language} />
-                  </Menu.Item>
+                  {
+                    !isMobileDevice && (
```
Comment (bbfsdev) [inline src/components/Layout/Layout.js:229]: No donation button in mobile?

**Review by bbfsdev — CHANGES_REQUESTED** [review]
image

On mute, volume should drop to zero, otherwise person thinks the volume is not muted.

**Review by bbfsdev — COMMENTED** [review]
http://bbdev6.kbb1.com:5501/he/lessons/cu/ntptfxdW?language=he&mediaType=video
image

image

This is related to chronicles.

File: public/locales/en/common.json (hunk 2 above)
Comment (davgur) [reply public/locales/en/common.json:192]: yes, I was add it on all lalnguages

File: src/components/Layout/Layout.js (hunk 3 above)
Comment (davgur) [reply src/components/Layout/Layout.js:229]: yes, before we was hide it with _mobile-hidden_ css

**Review by bbfsdev — APPROVED** [review]

**Pull request merged**